analytics/src/analytics/integrations/etldb/create_etl_db.sql

```diff
@@ -0,0 +1,96 @@
+CREATE TABLE IF NOT EXISTS gh_deliverable (
+	id SERIAL PRIMARY KEY,
+	ghid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	pillar TEXT, 
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+
+CREATE TABLE IF NOT EXISTS gh_deliverable_quad_map (
+	id SERIAL PRIMARY KEY,
+	deliverable_id INTEGER NOT NULL,
+	quad_id INTEGER,
+	d_effective DATE NOT NULL,
+	t_modified TIMESTAMP,
+	UNIQUE(deliverable_id, d_effective)
+);
+CREATE INDEX IF NOT EXISTS gh_dqm_i1 on gh_deliverable_quad_map(quad_id, d_effective);
+
+CREATE TABLE IF NOT EXISTS gh_epic (
+	id SERIAL PRIMARY KEY,
+	ghid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+
+CREATE TABLE IF NOT EXISTS gh_epic_deliverable_map (
+	id SERIAL PRIMARY KEY,
+	epic_id INTEGER NOT NULL,
+	deliverable_id INTEGER,
+	d_effective DATE NOT NULL,
+	t_modified TIMESTAMP,
+	UNIQUE(epic_id, d_effective)
+);
+CREATE INDEX IF NOT EXISTS gh_edm_i1 on gh_epic_deliverable_map(deliverable_id, d_effective);
+
+CREATE TABLE IF NOT EXISTS gh_issue (
+	id SERIAL PRIMARY KEY,
+	ghid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	type TEXT NOT NULL,
+	opened_date DATE,
+	closed_date DATE,
+	parent_issue_ghid TEXT,
+	epic_id INTEGER,
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+CREATE INDEX IF NOT EXISTS gh_issue_i1 on gh_issue(epic_id);
+
+CREATE TABLE IF NOT EXISTS gh_issue_history (
+	id SERIAL PRIMARY KEY,
+	issue_id INTEGER NOT NULL,
+	status TEXT,
+	is_closed INTEGER NOT NULL,
+	points INTEGER NOT NULL DEFAULT 0,
+	d_effective DATE NOT NULL,
+	t_modified TIMESTAMP,
+	UNIQUE(issue_id, d_effective)
```

Comment:
One thought on this and other tables -- it might be helpful to have a `t_created` column as well for debugging purposes.

That can be scoped into a future ticket though!

Reply:
Good suggestion, and done: [8946b80](https://github.com/HHS/simpler-grants-gov/pull/2617/commits/8946b80f04efa55f2cc4bd7b9364c90532485e33)